lms/djangoapps/instructor/access.py

```diff
@@ -72,15 +75,22 @@ def _change_access(course, user, level, action, send_email=True):
 
     NOTE: will create a group if it does not yet exist.
     """
-
     try:
         role = ROLES[level](course.id)
     except KeyError:
         raise ValueError("unrecognized level '{}'".format(level))
 
     if action == 'allow':
         if level == 'ccx_coach':
-            email_params = get_email_params(course, True)
+            # Try to using CCX course name instead of the course.
+            try:
+                ccx_course_object = CustomCourseForEdX.objects.get(pk=course.id.ccx)
```

Comment:
why do we need this query? Cannot we use incoming `course` argument? Is it a master course or CCX?

https://github.com/Livit/Livit.Learn.EdX/blob/95c64508b17aac821c9bee3100628bca2f236ef8/lms/djangoapps/ccx/utils.py#L313-L314

Reply:
How can I get the ccx if I don't use that query ? do you have any alternative for this?
I test using logger it return course name instead of ccx name so I assume it return a master course.

Reply:
@lukyfebrianto have you tried `course.display_name` ?

Reply:
@polesye I already tried it give me course name instead of ccx name and also it same with this method
https://github.com/Livit/Livit.Learn.EdX/blob/labster/develop/lms/djangoapps/instructor/enrollment.py#L300